modules/formulize/class/fileUploadElement.php

```diff
@@ -117,9 +117,10 @@ function loadValue($value, $ele_value, $element) {
     // $caption is the prepared caption for the element
     // $markupName is what we have to call the rendered element in HTML
     // $isDisabled flags whether the element is disabled or not so we know how to render it
-    // $element is the element object
+		// $element is the element object
     // $entry_id is the ID number of the entry where this particular element comes from
-    function render($ele_value, $caption, $markupName, $isDisabled, $element, $entry_id) {
```

Comment:
This and related function signature changes are simply bringing some element class files up to date with the current standard. We do pass the screen to the render method, though it is ignored in all standard elements, so I think that is only used in some custom element for ESAT or someone else.